onnxruntime/contrib_ops/cpu/cpu_contrib_kernels.cc

```diff
@@ -159,6 +159,7 @@ Status RegisterNchwcKernels(KernelRegistry& kernel_registry) {
   return Status::OK();
 }
 
+#if !defined(ORT_MINIMAL_BUILD)
```

Comment:
should this be `#if !defined(ORT_MINIMAL_BUILD) || defined(ORT_EXTENDED_MINIMAL_BUILD)` to be consistent with the kernel files?